src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -72,3 +74,119 @@ def test_storage_blob_azcopy_sync(self, resource_group, storage_account_info, te
             test_dir, container, storage_account))
         self.cmd('storage blob list -c {} --account-name {}'.format(
             container, storage_account), checks=JMESPathCheck('length(@)', 0))
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    @StorageTestFilesPreparer()
+    def test_storage_blob_azcopy_remove(self, resource_group, storage_account_info, test_dir):
```

Comment:
I think test_storage_remove() may be better because not only blob can be removed.

Reply:
test_storage_blob_azcopy_remove is focusing on blob remove. There is another test test_storage_file_azcopy_remove to cover file remove.